extensions/plugins/system/redcore/redcore.xml

```diff
@@ -122,6 +122,16 @@
                     <option value="0">JNO</option>
                 </field>
                 <field
+                        name="stateful_webservice"
+                        class="btn-group"
+                        type="radio"
+                        default="0"
```

Comment:
In the actual tracker item, @jatitoam requested default off for state, while I understand the desire to keep BC, stateful webservices by default really doesn't make sense to me in a RESTFUL API

Especially since any possible BC issue can be resolved with literally a click of a button.